tests/component/e2e/test_seeds.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
We have got small missunderstanding. This would be just a component test as it is using mock instead of real Flink, so should be in `test/component` in the future we should create an end-to-end test that will start actual Flink instance during test and it would be kept in `test/e2e`

Reply:
ok